### PR TITLE
Support HTTP/2 in Let's Encrypt

### PR DESCRIPTION
**Pull request by stanthetiger**
Modified the tlsConfig.NextProtos for Let's Encrypt and built-in HTTPS server in order to support HTTP/2.

### CODE REVIEW

File: modules/graceful/server.go [hunk 1]
```diff
@@ -106,7 +106,7 @@ func (srv *Server) ListenAndServe(serve ServeFunction) error {
 func (srv *Server) ListenAndServeTLS(certFile, keyFile string, serve ServeFunction) error {
 	config := &tls.Config{}
 	if config.NextProtos == nil {
-		config.NextProtos = []string{"http/1.1"}
+		config.NextProtos = []string{"h2"}
```
Comment (6543) [inline modules/graceful/server.go]: Why replace if you can add?

Comment (6543) [reply modules/graceful/server.go]:
```suggestion
		config.NextProtos = []string{"h2", "http/1.1"}
```

**Review by 6543 — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Comment by 6543** [comment]
@stanthetiger sorry we are in freeze and so asap v1.15.0-rc1 got released this will be merged :)

**Comment by stanthetiger** [comment]
ok, thanks 👍

**Review by zeripath — APPROVED** [review]

**Comment by zeripath** [comment]
I argue 1.15

**Comment by zeripath** [comment]
make lgtm work

**Pull request merged**